### PR TITLE
Sys 3752 benchmarks for new extrinsics

### PR DESCRIPTION
**Pull request by ivan-cholakov**
added benchmarks for `regenerate_lower_proof` and `set_lower_schedule_period`

### CODE REVIEW

**Review by nahuseyoum — APPROVED** [review]

**Review by fluorostani — APPROVED** [review]

File: pallets/token-manager/src/benchmarking.rs [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
 //! token manager pallet benchmarking.
 
-#![cfg(feature = "runtime-benchmarks")]
+// #![cfg(feature = "runtime-benchmarks")]
```
Comment (nahuseyoum) [inline pallets/token-manager/src/benchmarking.rs:21]: Do we want to comment this out?

File: pallets/token-manager/src/benchmarking.rs [hunk 2]
```diff
@@ -276,32 +282,10 @@ benchmarks! {
         lower.t1_recipient
     )
     verify {
-        assert_eq!(<T as pallet::Config>::Currency::free_balance(&lower.from_account_id), 0u32.into());
+        assert_eq!(<T as pallet::Config>::Currency::free_balance(&lower.from_account_id), 1000u32.into());
```
Comment (nahuseyoum) [inline pallets/token-manager/src/benchmarking.rs:285]: This check only verifies that no tokens have been burned but doesn't guarantee that scheduling has completed successfully. Can we check for the `LowerScheduled` event as well please?

File: pallets/token-manager/src/benchmarking.rs [hunk 3]
```diff
@@ -315,6 +299,32 @@ benchmarks! {
         assert_eq!(<T as pallet::Config>::Currency::free_balance(&treasury_account), amount.into());
         assert_eq!(<T as pallet::Config>::Currency::free_balance(&recipient), amount.into());
     }
+
+    regenerate_lower_proof {
+        let lower: Lower<T> = Lower::new().setup();
+        let params: LowerParams = BoundedVec::truncate_from(vec![
+        (BoundedVec::truncate_from(vec![0u8; 32]), BoundedVec::truncate_from(vec![1u8; 32]))]);
+        let lower_data = vec![0u8; 32]; 
+        // let requester = account("requester", 1, 1);
```
Comment (nahuseyoum) [inline pallets/token-manager/src/benchmarking.rs:308]:
```suggestion     
```

File: pallets/token-manager/src/benchmarking.rs [hunk 4]
```diff
@@ -315,6 +299,32 @@ benchmarks! {
...
+        let lower_proof_data = LowerProofData {
+            params: params.clone(),
+            abi_encoded_lower_data: BoundedVec::<u8, LowerDataLimit>::try_from(lower_data).expect("test"),
+        };
+
+        <LowersReadyToClaim<T>>::insert(lower.lower_id, lower_proof_data);
+    }: _(RawOrigin::<T::AccountId>::Signed(lower.from_account_id.clone()), lower.lower_id)
+    verify {
+        assert!(<LowersPendingProof<T>>::contains_key(lower.lower_id));
+        // assert_last_event::<T>(Event::<T>::RegeneratingLowerProof { lower_id, requester }.into());
```
Comment (nahuseyoum) [inline pallets/token-manager/src/benchmarking.rs:319]: I think we should check for this

**Review by alexpArtos — APPROVED** [review]

**Pull request merged**